notifications/tasks.py

```diff
@@ -1,12 +1,22 @@
 import logging
+from smtplib import SMTPException
 from celery import shared_task
 
 from django.core.mail import send_mail
 from django.contrib.auth import get_user_model
+from django.forms import ValidationError
 from django.utils import timezone
 
 from forum.settings import DEFAULT_FROM_EMAIL
-from .models import Notification, NotificationType
+from users.models import Role
+from investors.models import InvestorProfile
+from startups.models import StartUpProfile
+from .models import (
+    Notification,
+    NotificationType,
+    NotificationPreferences,
+    RolesNotifications
+)
 
 
```

Comment:
You should to add this line for coorect working  =>  logger = logging.getLogger('django')